vcd/sample_vcd_test_config.json

```diff
@@ -11,7 +11,14 @@
     "allowInsecure": true,
     "//": "tfAcceptanceTests is needed to allow the whole test suite to run",
     "//": "Warning: it may cost time (from 14 to 40 min) and possibly money",
-    "tfAcceptanceTests": false
+    "tfAcceptanceTests": false,
+    "//": "If enabled, the test suite will cache the provider connection for up to 20 minutes",
+    "//": "and then renew it automatically. This will save some heavy network traffic.",
+    "//": "The default is false",
+    "UseVcdConnectionCache": true,
+    "//": "The default for max retry timeout is 60, which often leads to premature test termination",
+    "//": "If no value is provided, the test suite will set the value to 300",
```

Comment:
Really confusing sentence about defaults. Imagine new user reading: ... "default for max retry timeout is 60" ... "the test suite will set the value to 300" ...
I suggest amending the comment that it's clear that 60 is default for _Terraform provider_, while tests set a higher default.

Reply:
fixed